README.md

```diff
@@ -79,6 +79,37 @@ Counterexample: [p_price_uint96 = 39614081294025656978550816768, p_quantity_uint
 
 _(In this specific example, Halmos discovered an input that violated the assertion, which was missed by the fuzzer!)_
 
+## Develop
+
+```sh
+# if you want to submit a pull request, fork the repository:
+
+gh repo fork a16z/halmos
+
+# or if you just want to develop locally, clone it
+# git clone git@github.com:a16z/halmos.git
+
+# create and activate a virtual environment
+
+python3.10 -m venv .venv
```

Comment:
3.10 doesn't seem representative.  we support >=3.8 and the latest is 3.11.  can we just say `python3`?

Reply:
let's recommend 3.11 then, makes more sense to me